js/blocks/controls/index.js

```diff
@@ -1,21 +1,24 @@
-import BlockLabTextControl from './text'
-import BlockLabTextareaControl from './textarea'
-import BlockLabRichTextControl from './rich-text'
-import BlockLabURLControl from './url'
-import BlockLabEmailControl from './email'
-import BlockLabNumberControl from './number'
-import BlockLabColorControl from './color'
-import BlockLabImageControl from './image'
-import BlockLabCheckboxControl from './checkbox'
-import BlockLabRadioControl from './radio'
-import BlockLabRangeControl from './range'
-import BlockLabSelectControl from './select'
-import BlockLabMultiselectControl from './multiselect'
-import BlockLabPostControl from './post'
-import BlockLabRepeaterControl from './repeater'
-import BlockLabTaxonomyControl from './taxonomy'
-import BlockLabToggleControl from './toggle'
-import BlockLabUserControl from './user'
+/**
+ * Internal dependencies
+ */
```

Comment:
This DocBlock and the comma at the end of the file look to be the only diff in this file, though there's a lot more red and green than that.